src/fragments/lib/push-notifications/ios/getting_started/apns-cli-resources.mdx

```diff
@@ -17,4 +25,17 @@ Choose _Certificate_ when promoted:
 Key
 ```
 
-The CLI will prompt for your _p12 certificate path_, enter it respectively.
+The CLI will prompt for your _p12 certificate path_, enter a path relative to the location where you ran the command.
+
+
+<Callout>
+
+Note: If you receive this error:
+```console 
+🛑 Command failed: openssl pkcs12 -in [path_to_your_cert].p12 -out [output_path].pem -nodes -passin pass:
+Error outputting keys and certificates
+```
+
+Please try using LibreSSL 3.3.6 instead of OpenSSL.
```

Comment:
Should we also put the related issue here for the updates? https://github.com/aws-amplify/amplify-cli/issues/12969

p.s. This was one of the most annoying errors that I had.